packages/react/src/components/Timeline/TimelineItem/TimelineItem.tsx

```diff
@@ -0,0 +1,164 @@
+  isBottom?: boolean;
+  weight?: TimelineItemProps['weight'];
+};
+
+const useUtilityClasses = (ownerState: TimelineItemOwnerState) => {
+  const { classes, weight } = ownerState;
+
+  const slots = {
+    root: ['root', `weight${weight}`],
+    oppositeContent: ['oppositeContent']
+  };
+
+  return composeClasses(slots, getTimelineItemUtilityClass, classes);
+};
+
+const TimelineItemRoot = styled('div', {
+  name: 'ESTimelineItem',
+  slot: 'Root',
+  overridesResolver: (props, styles) => styles.root
+})<{ ownerState: TimelineItemOwnerState }>(() => ({
+  display: 'flex',
+  alignItems: 'center',
+  gap: '12px',
+  padding: '8px 24px',
+  position: 'relative'
+}));
+
+const TimelineItemContent = styled('div', {
+  name: 'ESTimelineItem',
+  slot: 'Content',
+  overridesResolver: (props, styles) => styles.root
+})(() => ({
+  display: 'flex',
+  alignItems: 'center',
+  gap: '16px'
+}));
+
+const TimelineItemContainer = styled('div', {
+  name: 'ESTimelineItem',
+  slot: 'Container',
+  overridesResolver: (props, styles) => styles.root
+})(() => ({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: '4px'
+}));
+
+const TimelineItemHeader = styled('div', {
+  name: 'ESTimelineItem',
+  slot: 'Header',
+  overridesResolver: (props, styles) => styles.root
+})(({ theme }) => ({
+  ...theme.typography.caption,
+  color: theme.palette.monoA.A600
+}));
+
+const TimelineItemOppositeContent = styled('div', {
+  name: 'ESTimelineItem',
+  slot: 'OppositeContent',
+  overridesResolver: (props, styles) => styles.root
+})<{ ownerState: TimelineItemOwnerState }>(({ theme, ownerState }) => ({
+  ...theme.typography.caption,
+  color: theme.palette.monoA.A600,
+  width: '56px',
+  flexShrink: 0,
+  display: 'flex',
+  justifyContent: 'flex-end',
+
+  ...(ownerState.weight === 'sm' && {
+    ...theme.typography.body100Bold,
+    color: theme.palette.monoA.A700
+  })
+}));
+
+const TimelineItemPoint = styled('div', {
+  name: 'ESTimelineItem',
+  slot: 'DividerPoint',
+  overridesResolver: (props, styles) => styles.root
+})<{ ownerState: TimelineItemOwnerState }>(({ theme, ownerState }) => ({
+  height: '5px',
+  width: '5px',
+  borderRadius: '50%',
+  background: theme.palette.monoA.A500,
+  ...(ownerState.weight === 'sm' && {
+    height: '7px',
+    width: '7px',
+    background: theme.palette.monoA.A600
+  })
+}));
+
+const TimelineItemLine = styled('div', {
+  name: 'ESTimelineItem',
+  slot: 'DividerLine',
+  overridesResolver: (props, styles) => styles.root
+})<{ ownerState: TimelineItemLineOwnerState }>(({ ownerState, theme }) => ({
+  height: 'calc(50% - 6px)',
+  width: '1px',
+  background: theme.palette.monoA.A100,
+  position: 'absolute',
+  left: '94px',
+  top: 'calc(50% + 6px)',
+  ...(ownerState.isTop && {
+    top: 0
+  }),
+  ...(ownerState.weight === 'sm' && {
+    left: '95px'
+  })
+}));
+
+const TimelineItemDivider = styled('div', {
+  name: 'ESTimelineItem',
+  slot: 'Divider',
+  overridesResolver: (props, styles) => styles.root
```

Comment:
```suggestion
  overridesResolver: (props, styles) => styles.divider
```
И у остальных проверить